src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/AddressDtoValidatorTest.java

```diff
@@ -120,13 +117,37 @@ void testErrorReportedWhenServiceAddressIsNull() {
         String validationMessage = String.format(ValidationMessages.NOT_NULL_ERROR_MESSAGE, EntityDto.SERVICE_ADDRESS_FIELD);
         assertError(parentField, "", validationMessage, errors);
     }
+    @Test
```

Comment:
I don't think you've added this, have you? It's already in the test class so presume it's GitHub getting confused...

Reply:
No it had to be added. The parameterised test annotations had been removed and there were unused imports that had to be removed as well.

Reply:
Ah, now I see.